database/tests/database_test.go

```diff
@@ -4,34 +4,72 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"path"
 	"reflect"
 	"testing"
 
 	"github.com/thrasher-corp/gocryptotrader/database"
 	"github.com/thrasher-corp/gocryptotrader/database/drivers"
-	dbpsql "github.com/thrasher-corp/gocryptotrader/database/drivers/postgres"
-	dbsqlite "github.com/thrasher-corp/gocryptotrader/database/drivers/sqlite"
+
+	psqlConn "github.com/thrasher-corp/gocryptotrader/database/drivers/postgres"
+	sqliteConn "github.com/thrasher-corp/gocryptotrader/database/drivers/sqlite"
 )
 
 var (
-	tempDir string
+	tempDir              string
+	postgresTestDatabase *database.Config
+)
+
+func getConnectionDetails() *database.Config {
+	_, exists := os.LookupEnv("TRAVIS")
+	if exists {
+		return &database.Config{
+			Enabled: true,
+			Driver:  "postgres",
+			ConnectionDetails: drivers.ConnectionDetails{
+				Host:     "localhost",
+				Port:     5432,
+				Username: "postgres",
+				Password: "",
+				Database: "gct_dev_ci",
+				SSLMode:  "",
+			},
+		}
+	}
 
-	postgresTestDatabase = database.Config{
+	_, exists = os.LookupEnv("APPVEYOR")
+	if exists {
+		return &database.Config{
+			Enabled: true,
+			Driver:  "postgres",
+			ConnectionDetails: drivers.ConnectionDetails{
+				Host:     "localhost",
+				Port:     5432,
+				Username: "postgres",
+				Password: "Password12!",
+				Database: "gct_dev_ci",
+				SSLMode:  "",
+			},
+		}
+	}
+
+	return &database.Config{
 		Enabled:           true,
 		Driver:            "postgres",
 		ConnectionDetails: drivers.ConnectionDetails{
-			//Host:     "",
+			//Host:     "localhost",
 			//Port:     5432,
-			//Username: "",
+			//Username: "gct",
 			//Password: "",
-			//Database: "",
+			//Database: "gct-dev",
 			//SSLMode:  "",
```

Comment:
Could probably rm these comments

Reply:
I originally had it empty but decided to leave them there just to let users know what details are expected if they want to do postgres testing

ideally i would have had all the database config details in the configtest.json file but with multi-database testing deciding what one to default was a pain and confusing for users